delete-user-data/functions/package.json

```diff
@@ -17,19 +17,19 @@
   "author": "Lauren Long <longlauren@google.com>",
   "license": "Apache-2.0",
   "dependencies": {
+    "@google-cloud/pubsub": "^3.3.0",
```

Comment:
Is this dependency required, it doesn't appear to have existed previously?

`"@google-cloud/pubsub"`

Reply:
Interesting, ok sounds good!